sdk/keyvault/keyvault-certificates/src/core/keyVaultClientContext.ts

```diff
@@ -11,7 +11,7 @@
 import * as coreHttp from "@azure/core-http";
 
 const packageName = "@azure/keyvault-certificates";
-const packageVersion = "4.0.0-preview.8";
+const packageVersion = "4.0.0-preview.9";
```

Comment:
@ramya-rao-a do we know why there are 2 constants in each of the keyvault libraries that has the package version?